bpm/bonita-core/bonita-process-engine/src/test/java/org/bonitasoft/engine/api/impl/transaction/actor/ExportActorMappingTest.java

```diff
@@ -0,0 +1,129 @@
+/*******************************************************************************
+ * Copyright (C) 2015 BonitaSoft S.A.
+ * BonitaSoft, 32 rue Gustave Eiffel - 38000 Grenoble
+ * This library is free software; you can redistribute it and/or modify it under the terms
+ * of the GNU Lesser General Public License as published by the Free Software Foundation
+ * version 2.1 of the License.
+ * This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth
+ * Floor, Boston, MA 02110-1301, USA.
+ ******************************************************************************/
+
+package org.bonitasoft.engine.api.impl.transaction.actor;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.StringReader;
+import java.io.StringWriter;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.Marshaller;
+import javax.xml.bind.Unmarshaller;
+
+import org.bonitasoft.engine.bpm.bar.ActorMappingConverter;
+import org.bonitasoft.engine.bpm.bar.actorMapping.Actor;
+import org.bonitasoft.engine.bpm.bar.actorMapping.ActorMapping;
+import org.junit.Test;
+
+/**
+ * Created by mazourd on 13/07/15.
+ */
+public class ExportActorMappingTest {
```

Comment:
We use to name our tests like that
```java
public void sould_do_something_when_something() 
```
Furthermore, we follow the AAA pattern http://c2.com/cgi/wiki?ArrangeActAssert

All `System.out.print` need to be removed
Globally, this test suffer from code duplication and need to be refactored